app/cli/cmd/attestation_add.go

```diff
@@ -138,3 +147,54 @@ func newAttestationAddCmd() *cobra.Command {
 
 	return cmd
 }
+
+// displayMaterialInfo prints the material information in a table format.
+func displayMaterialInfo(status *action.AttestationStatusMaterial, policyEvaluations []*action.PolicyEvaluation) error {
+	if status == nil {
+		return nil
+	}
+
+	mt := newTableWriter()
+
+	mt.AppendRow(table.Row{"Name", status.Material.Name})
+	mt.AppendRow(table.Row{"Type", status.Material.Type})
+	mt.AppendRow(table.Row{"Required", hBool(status.Required)})
+
+	if status.IsOutput {
+		mt.AppendRow(table.Row{"Is output", "Yes"})
+	}
+
+	if status.Value != "" {
+		v := status.Value
+		if status.Tag != "" {
+			v = fmt.Sprintf("%s:%s", v, status.Tag)
+		}
+		mt.AppendRow(table.Row{"Value", wrap.String(v, 100)})
+	}
+
+	if status.Hash != "" {
+		mt.AppendRow(table.Row{"Digest", status.Hash})
+	}
+
+	if len(status.Material.Annotations) > 0 {
+		mt.AppendRow(table.Row{"Annotations", "------"})
+		for _, a := range status.Material.Annotations {
+			value := a.Value
+			if value == "" {
+				value = NotSet
+			}
+			mt.AppendRow(table.Row{"", fmt.Sprintf("%s: %s", a.Name, value)})
+		}
+	}
+
+	mt.AppendRow(table.Row{"Policy evaluations", "------"})
+	if len(policyEvaluations) == 0 {
```

Comment:
This is incorrect. As you can see [here](https://github.com/chainloop-dev/chainloop/blob/b42f2b8d479952daf4e5d6fcda14d12503766d0b/app/cli/cmd/workflow_workflow_run_describe.go#L232), not having violations doesn't mean skipped, it could, but not always.

Instead of us coming up with the rendering of the policies, could we use that function instead? Note that's in the same package.